### ISSUE TITLE
Corrupt compression package

### ISSUE THREAD

**Issue by yunchao-ding** [majedelhelou/VIDIT#10]
Hi, why after I downloaded the compressed package of "one2one" from "Getting the CVPR 2021 NTIRE data (with depth maps)" here, when I decompressed "Train", I was prompted that the compressed file was damaged.

**Comment by GinaZhan** [comment]
Do you get any solution? We hit the same problem.

**Comment by CA2528357431** [comment]
yes, the same issue. The package is corrupted and only part of the data could be achieved

**Comment by majedelhelou** [comment]
I just ran into the same issue, even though the data never changed. The issue is likely coming from the hosting, an issue was resolved over a year ago but there seems to be something else again. 

You can reach the support team of the hosting platform here: [EPFL IC IT](https://www.epfl.ch/schools/ic/it/en/organisation-du-support-informatique/), email: support-icit@epfl.ch
Connected lab at EPFL: IVRL

Hope the team will be able to sort it out for you quickly!

**Comment by jayhsu0627** [comment]
same issue
